### PR TITLE
brian's Data::Rx article

### PR DESCRIPTION
**Pull request by briandfoy**
Here's the first draft

### CODE REVIEW

File: 2024/incoming/santas-reindeer.pod [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+
+=begin code
+
+	{
+	"Name": "Donner",
+	"aliases": [ "Dunder", "Donder" ],
+	"start-date": "1823-12-24"
+	}
+
+=end code
```
[the remark this answers is not in the document]
Comment (briandfoy) [reply 2024/incoming/santas-reindeer.pod]: I added a few hints. Let's see if they work. If they do, my PR for `new_article` can be merged (#446)

**Comment by oalders** [comment]
Is this ready for final review?

**Comment by oalders** [comment]
Is this ready for final review?

**Comment by briandfoy** [comment]
The article is about 75% there. I'll finish it up by next Monday.

**Comment by oalders** [comment]
Based on commit messages, this one is ready for review?

**Comment by briandfoy** [comment]
Yes, review away. Edit freely. I'll read it again later.

**Review by openstrike — CHANGES_REQUESTED** [review]
Just some minor fixes for grammar and typos. The article reads well. I have not tested the code/configs.

**Comment by briandfoy** [comment]
All suggestions applied. Thanks for the work!

**Review by openstrike — APPROVED** [review]

**Comment by oalders** [comment]
Thanks @briandfoy and @openstrike!

**Pull request merged**